scripts/run_project_tests.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+import subprocess
+import argparse
+import sys
+import os
+from typing import Dict, List
+
+import yaml
+from colorama import Fore, init
+
+init(autoreset=True)
+
+"""
+A script to run unity and pytest tests in a host environment for Phoenix-RTOS.
+
+Example of config file:
+
+    campaign_name:
+        unity:
+            - binary1 args
+            - binary2
+        pytest:
+            - path1 args
+            - path2
+
+NEED TO KNOW:
+
+    Need vm.map_rnd_bits set to 28
+    You can use: sudo sysctl vm.mmap_rnd_bits=28
+    due to the issue reported here: https://github.com/phoenix-rtos/phoenix-rtos-project/issues/1032
+"""
+
+
+class TestSetupError(Exception):
+    pass
+
+
+def print_fancy(message: str, color: str, flush: bool = False, end: str = "\n") -> None:
```

Comment:
so now you can use `print` directly as there is nothing fancy about this function :)